manifests/qpid.pp

```diff
@@ -18,12 +18,39 @@
     require   => Service['qpidd'],
     logoutput => true,
   } ~>
-  exec { 'bind katello entitlements queue to qpid exchange messages that deal with entitlements':
-    command   => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' bind event ${candlepin_event_queue} '*.*'",
+  exec { 'bind katello entitlements queue to qpid exchange messages that deal with entitlement creation':
+    command   => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' bind event ${candlepin_event_queue} entitlement.created",
+    onlyif    => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' queues ${candlepin_event_queue}",
```

Comment:
This is a check to see whether the command should be executed, is there a more specific check we could use to check the binding?